vllm/model_executor/model_loader/veturboio.py

```diff
@@ -0,0 +1,119 @@
+import os
+import time
+from dataclasses import dataclass
+from typing import Optional, Tuple
+
+import torch
+from torch import nn
+
+from vllm.config import ModelConfig, ParallelConfig
+from vllm.logger import init_logger
+
+veturboio_error_msg = None
+
+try:
+    import veturboio
+    from veturboio.ops.load_utils import IOHelper
+    helper = None if IOHelper is None else veturboio.init_io_helper()
+except ImportError as e:
+    veturboio_error_msg = str(e)
+
+logger = init_logger(__name__)
+
+
+@dataclass
+class VeturboIOConfig:
+    model_files: Optional[Tuple[str, os.PathLike]] = None
+    map_location: Optional[str] = "cpu"
+    enable_fast_mode: Optional[bool] = True
+    num_thread: Optional[int] = 32
+    use_pinmem: Optional[bool] = False
+    use_direct_io: Optional[bool] = False
+    use_cipher: Optional[bool] = False  # not implemented yet
```

Comment:
what is usage of `cipher` here?

Reply:
When loading the model on SFCS, veturbio can control whether encryption and decryption are enabled through the `use_cipher` flag. But we haven't tested loading model files on SFCS yet, so I commented 'not implemented yet' after this option.

Reply:
make sense. technically, it should apply to any encrypted model. let's leave it here with comment. It looks good to me